ksqldb-engine/src/test/java/io/confluent/ksql/engine/KsqlEngineTest.java

```diff
@@ -135,7 +138,6 @@ public class KsqlEngineTest {
 
   @Before
   public void setUp() {
-    sharedRuntimeEnabled.put(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED, true);
```

Comment:
reassignment on line 143 so this doesn't make sense.
`sharedRuntimeDisabled.put(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED, false);`